core/domain/exp_services_test.py

```diff
@@ -2391,6 +2399,7 @@ class ExplorationConversionPipelineTests(ExplorationServicesUnitTests):
       id: EndExploration
     param_changes: []
   %s:
+    classifier_model_id: null
```

Comment:
When we add new migrations, we also add a new test to ensure that specific version migrates correctly. See the tests in this file for an example on this and see the corresponding commit (recommend using git blame on GitHub for finding a commit).